lib/dialog_processor.py

```diff
@@ -84,7 +84,7 @@ def save_index_to_tokens(index_to_token, token_index_path):
 def get_index_to_token(token_index_path):
     with codecs.open(token_index_path, 'r', 'utf-8') as token_index_fh:
         index_to_token = json.load(token_index_fh)
-        index_to_token = {int(k): v for k, v in index_to_token.iteritems()}
+        index_to_token = {int(k): v for k, v in index_to_token.items()}
```

Comment:
And iteriterms is not defined in python 3. Is replaced items()
Dňa 16.3.2016 7:27 PM používateľ "Nicolas" <notifications@github.com>
napísal:

> In lib/dialog_processor.py
> <https://github.com/nicolas-ivanov/debug_seq2seq/pull/8#discussion_r56389221>
> :
>
> > @@ -84,7 +84,7 @@ def save_index_to_tokens(index_to_token, token_index_path):
> >  def get_index_to_token(token_index_path):
> >      with codecs.open(token_index_path, 'r', 'utf-8') as token_index_fh:
> >          index_to_token = json.load(token_index_fh)
> > -        index_to_token = {int(k): v for k, v in index_to_token.iteritems()}
> > +        index_to_token = {int(k): v for k, v in index_to_token.items()}
>
> why to prefer items() over iteriterms() ?
> iteriterms() seems to be more computationally efficient
>
> http://stackoverflow.com/questions/10458437/what-is-the-difference-between-dict-items-and-dict-iteritems
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/nicolas-ivanov/debug_seq2seq/pull/8/files/5215c1cf85e88f4171280142adeb321c82c40db1#r56389221>
>